csrc/mmdeploy/codebase/mmaction/format_shape.cpp

```diff
@@ -127,6 +130,14 @@ Result<void> FormatShape::Apply(Value& data) {
 
 MMDEPLOY_REGISTER_TRANSFORM(FormatShape);
 
-MMDEPLOY_DEFINE_REGISTRY(FormatShapeOp);
+MMDEPLOY_DEFINE_REGISTRY(FormatShapeImpl);
```

Comment:
Why not merge `FormatShapeImpl` and `FormatShape`?
BTW, I think it's better to open another PR to refactor mmaction's format shape.
It can reduce review workload